style.css

```diff
@@ -85,14 +86,45 @@ li {
   font-weight: 500;
 }
 
-#nav_menu {
+.hidden {
   display: none;
 }
 
+.overflow {
+  overflow: hidden;
+}
+
+.toggled-menu {
+  background: #fff;
+  display: block;
+  position: absolute;
+  top: -24px;
+  left: 0;
+  height: 100vh;
+  padding-top: 100px;
+  padding-left: 24px;
+  z-index: 0;
+  border-radius: 0 0 48px 48px;
+}
+
+.nav-link {
+  color: #3a4a42;
+  display: block;
+  font-size: 32px;
+  font-weight: 600;
+  margin-bottom: 40px;
+
+  /* border: 2px solid red; */
+}
+
 /* i added this to remove some white space that appeared
 after i put the burger menu inside a navbar */
 #burger-menu {
```

Comment:
Kindly delete commented code that has no use anymore for it makes your work look dirty and unprofessional. And it would be nice to add a line-height of 44px to match the gap between the nav-links in the template its really important to pay attention to detail. Please don't feell like Im being unfair to you Im only doing this so your work stands out.